src/coffee/directives/api/utils/model-key.coffee

```diff
@@ -135,4 +140,16 @@ angular.module('uiGmapgoogle-maps.directives.api.utils')
           return
         #set isInit to true to force redraw after all updates are processed
         child.updateModel model
+
+
+      modelsLength: =>
```

Comment:
cached modelsLength for dealing with lengths of Array and Objects a little quicker without needing to evaluate isArray and Object.keys as much.